### PR TITLE
EWL-4914: Add range field slider

### PR DESCRIPTION
**Pull request by designerneil**
## Ticket(s)

**Jira Ticket**
- [EWL-4914: SG2 | Create "Form step" atom and variants](https://issues.ama-assn.org/browse/EWL-4914)

## Description
Add form step atom with variants which include completed but can't be edited,  completed but can be edited, editable, error

## To Test
- [ ] `gulp serve`
- [ ] visit http://localhost:3000/?p=atoms-form-step
- [ ] observe a 5 steps: completed, editable, current, error, editable

## Visual Regressions
backstop tested
No failures noted

## Relevant Screenshots/GIFs
screen shot 2018-04-27 at 2 53 58 pm

## Remaining Tasks
N/A

## Additional Notes
N/A
---

[Guidelines for Contribution](CONTRIBUTING.md)
[SG2 Standards](ama-style-guide-2/docs/standards.md)

### CODE REVIEW

**Comment by designerneil** [comment]
@labbydev this PR has been review with @toph from UX. He will update the zeplin doc to reflect the changes we made in the UX collab meeting

File: styleguide/source/assets/js/form-items.js [hunk 1]
```diff
@@ -30,6 +29,27 @@
...
+          var handle = $( "#currentValue" );
+
+          $(".ama__range-field").slider({
+            animate: true,
+            range: "min",
+            value: 1,
+            min: 1,
+            max: 100,
+            step: 1,
+            create: function(){
```
Comment (nstriedinger) [inline styleguide/source/assets/js/form-items.js]: Add a space after keyword function()

File: styleguide/source/assets/js/form-items.js [hunk 2]
```diff
@@ -30,6 +29,27 @@
           $('.textarea').keyup(function() {
             count_remaining_character();
           });
+
+          // Range Field
+          var legend = $('.ama__range-field__legend');
+          var handle = $( "#currentValue" );
```
Comment (nstriedinger) [inline styleguide/source/assets/js/form-items.js:35]: Use single quotes and remove spaces. See above code.

File: styleguide/source/assets/scss/01-atoms/_range-slider.scss [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+    justify-content: space-between;
+    flex-direction: row;
+  }
+
+  &__legend li {
+    list-style: none;
+    margin: 0;
+  }
+
+  .valuebox {
```
Comment (nstriedinger) [inline styleguide/source/assets/scss/01-atoms/_range-slider.scss]: This doesn't look like it goes here. Should it be `&__valuebox`?

File: styleguide/source/assets/js/form-items.js (hunk 2 above)
Comment (designerneil) [reply styleguide/source/assets/js/form-items.js:35]: Yeah ur right

**Pull request merged**